Game/Audio/Audio.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.Xna.Framework.Media;
+using Microsoft.Xna.Framework.Audio;
+using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection.Metadata;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+
+namespace MainGame.Audio
+{
+    public class AudioManager
```

Comment:
Definitely not needed for our current application but in the long run it'll be better to have an audio manager interface or an audio interface. Inside the interface can just have the "PlaySFX" and "Mute" functions since everything that uses the AudioManager only really needs to be aware of those function, it doesn't need to be aware of the rest of this class. It'll also make swapping to a different AudioManager much easier.